style.css

```diff
@@ -64,8 +64,9 @@ body, html {
   
   .imageAboutPage {
     animation-name: move;
-    animation-duration: 5s;
+    animation-duration: 2s;
```

Comment:
Wow! 👍, Nice using the transform. I think using duplicate styling will affect the layout of your document. Could you consider removing the duplicates?